src/clj/game/cards/operations.clj

```diff
@@ -239,6 +239,14 @@
     :msg "do 7 meat damage"
     :effect (effect (damage eid :meat 7 {:card card}))}
 
+   "Building Blocks"
+   {:choices {:req #(and (= (:side %) "Corp")
+                         (ice? %)
```

Comment:
Technically the card only specified it has to be a Barrier, that it is also a piece of ICE is an assumption on our part.